Dockerfile

```diff
@@ -25,7 +25,7 @@ ARG GIT_COMMIT_SHA
 ENV transformer_build_version=$version
 ENV git_commit_sha=$GIT_COMMIT_SHA
 COPY package*.json ./
-RUN npm install
+RUN npm ci
```

Comment:
Should we omit dev dependencies?

Reply:
That's a good thought. 
Might work for now but there are two problems:
- we can't run tests on the image and other scripts that need dev dependencies
- They are essential for building the image when we migrate to TypeScript